corehq/apps/hqadmin/static/hqadmin/js/project_map.js

```diff
@@ -101,7 +101,8 @@ var projectMapInit = function(mapboxAccessToken) {
             var self = this;
             self.selectedCountry = ko.observable('country name');
             self.selectedProject = ko.observable('project name');
-            self.tableProperties = ['Sector', 'Sub-Sector', 'Active Users', 'Countries'];
+            self.tableProperties = ['Name', 'Organization', 'Notes', 'Sector', 'Sub-Sector', 'Active Users', 'Countries'];
+            self.externalTableProperties = ['Sector', 'Sub-Sector', 'Active Users', 'Countries'];
```

Comment:
Might want to name the former `internalTableProperties` (or otherwise namespace it) to make it explicit that it's only to be used in the internal mode.